src/components/UpgradePlanForm/UpgradePlanForm.js

```diff
@@ -98,7 +98,9 @@ class UpgradePlanForm extends React.Component {
                     <li
                       className={
                         'field-item' +
-                        (touched[fieldNames.message] && errors[fieldNames.message] ? ' error' : '')
+                        (submitCount && touched[fieldNames.message] && errors[fieldNames.message]
```

Comment:
I like this magic submitCount that comes with formik. 😄

Reply:
Haha, it is not magic: all changes are explicit and no convention based.